code/_onclick/telekinesis.dm

```diff
@@ -173,6 +173,9 @@
 		apply_focus_overlay()
 		//Only items can be thrown 10 tiles everything else only 1 tile
 		focus.throw_at(target, focus.tk_throw_range, 1,user)
+		var/turf/start_turf = get_turf(focus)
+		var/turf/end_turf = get_turf(target)
+		user.log_message("has thrown [focus] from [AREACOORD(start_turf)] to [AREACOORD(end_turf)] using Telekinesis", LOG_ATTACK)
```

Comment:
```suggestion
		user.log_message("has thrown [focus] from [AREACOORD(start_turf)] towards [AREACOORD(end_turf)] using Telekinesis", LOG_ATTACK)
```

call it nitpicking but i think this improves legibility of this log wherein someone may not realize it's being thrown *towards* the target, not necessarily where it will end up